tests/unit/amo/components/Errors/TestErrorComponent.js

```diff
@@ -13,7 +13,6 @@ describe(__filename, () => {
   it('renders an ErrorComponent', () => {
     const root = render();
 
-    expect(root.find(NestedStatus)).toHaveProp('code', 400);
```

Comment:
@bobsilverberg I have enforce the set of valid values for code. Do we require this test case ?

Reply:
No, we don't need a test for a default value anymore as there is no default.